compute/instances/troubleshooting/fix-dns-routed-ipv6-only-debian-bullseye.mdx

```diff
@@ -0,0 +1,143 @@
+---
+meta:
+  title: Fixing DNS resolution with a routed IPv6-only setup on Debian Bullseye
+  description: This page helps you to fix DNS resolution with a routed IPv6-only setup on Debian Bullseye
+content:
+  h1: Fixing DNS resolution with a routed IPv6-only setup on Debian Bullseye
+  paragraph: This page helps you to fix DNS resolution with a routed IPv6-only setup on Debian Bullseye
+tags: dns ipv6 Instance debian bullseye
+dates:
+  validation: 2024-01-23
+  posted: 2024-01-23
+categories:
+  - compute
+---
+
+This guide outlines the steps to enable DNS resolution on a Scaleway Instance that uses the Debian Bullseye image, configured with one or more routed IPv6 addresses while lacking any IPv4 addresses.
```

Comment:
> while lacking any IPv4 addresses.

I think that the wording is misleading, because the verb _to lack_ suggests that it should be there.

```suggestion
This guide outlines the steps to enable DNS resolution on a Scaleway Instance that uses the Debian Bullseye image, configured with one or more routed IPv6 addresses and without any IPv4 addresses.
```